src/app/components/introduction.tsx

```diff
@@ -1,3 +1,6 @@
+import Image from 'next/image';
+import profilePicture from "../../../public/profile-picture.jpg";
```

Comment:
🌵 I noticed your quotes are sometimes inconsistent as to whether they're single or double quotes, which suggests you might not have [Prettier](https://prettier.io/) or another code formatter set up. 

I'd recommend doing that at some point so that you can have consistent formatting.

Reply:
Ahh good catch. I think I do have Prettier installed, but maybe it's not set up correctly or strict enough to catch these things. I'll look into it!

Reply:
Yeah you might just need to add the config to your package.json or something. I usually use 
```json
        "prettier": { 
                 "useTabs": true, 
                 "tabWidth": 4, 
                 "semi": true, 
                 "singleQuote": true, 
                 "trailingComma": "all" 
         },
```

Reply:
Done! Thanks for those settings